proposed/links.md

```diff
@@ -162,14 +181,92 @@ interface LinkInterface
 }
 ~~~
 
-#### 3.2.1 `Psr\Http\Link\LinkCollectionInterface`
+### 3.2 `Psr\Http\Link\EvolvableLinkInterface`
+
+~~~php
+<?php
+
+namespace Psr\Http\Link;
+
+/**
+ * An evolvable link value object.
+ */
+interface EvolvableLinkInterface extends LinkInterface
+{
+    /**
+     * Returns an instance with the specified href.
```

Comment:
Per discussion on IRC, leave as-is; we used this verbiage specifically in PSR-7 due to late feedback that indicated that "a new instance" would be too narrowly interpreted as not allowing cloning.